### ISSUE TITLE
[Feature request] Combine docker containers for those with non-standard Docker configurations

### ISSUE THREAD

**Issue by RocketSEA** [tubearchivist/tubearchivist#279]
While I understand that the standard practice for docker is to split out individual services into separate containers, sometimes this causes more problems than it's worth. Is there any possibility of also including built images combining all of the services? This would drastically simplify my workflow for my specific configuration. The only docker containers I have issues with are the ones that split out dependencies into separate containers (probably due to user error) and I'm sure I'm not alone.

**Comment by RocketSEA** [comment]
Of course, I'm not trying to say to drop the separate containers entirely.

**Comment by pairofcrocs** [comment]
It's something that has been discussed, but ultimately, at this time we're going to keep them separated.

It's a logistical nightmare combining, maintaining, and building three separate containers into open application.

If you have any issues setting it up, feel free to hop over to Discord where we'd be more than willing to guide you through the installation!

**Comment by bbilly1** [comment]
I know, Docker has a learning curve, but it's very much a worthwhile investment. It opens so many doors in the world of selfhosting. 

Combining the three containers into one, as mentioned, would be a big hassle to do, very hard to debug if any errors arise and you would end up with one huge container. 

There are lots of helpful projects out there to deploy a stack of container if you don't have access to regular docker-compose, Portainer is a popular one.
